sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/implementation/caches/RxCollectionCache.java

```diff
@@ -117,19 +128,20 @@ public void refresh(String resourceAddress, Map<String, Object> properties) {
 
                     });
         }
-        return Mono.empty();
+        return Mono.just(new Utils.ValueHolder<>(null));
```

Comment:
I think ValueHolder has initialize method to create new object

Reply:
Yes it does, in fact I used that initially, but it creates more confusion as if it was initializing something big. I just wanted to make it clear and concise, so used constructor :)